ml-agents/mlagents/model_serialization.py

```diff
@@ -18,6 +21,11 @@
 from tensorflow.python.framework import graph_util
 from mlagents.trainers import tensorflow_to_barracuda as tf2bc
 
+if LooseVersion(tf.__version__) < LooseVersion("1.12.0"):
```

Comment:
Should we also filter out tf 2.x ?

Reply:
I don't want to explicitly filter it out. A future version of tf2onnx should support f2 2.x, in which case the `import` should succeed.